vm/vmgs/vmgs/src/vmgs_impl.rs

```diff
@@ -1731,41 +1709,29 @@ pub mod save_restore {
         #[derive(Protobuf)]
         #[mesh(package = "vmgs")]
         pub struct SavedBlockStorageMetadata {
+            /// The byte capacity. Redundant with sector_count * sector_size.
```

Comment:
is there a way to rewrite / mark the redundant fields as redundant here? i.e: adding some underscores, moving them into one block at the top, removing `pub`, something like that?

Reply:
I don't think so. Someone still has to fill out the redundant fields, so making them look weird doesn't really do anything. Maybe I could see if there are any fields the old code doesn't even look at and just remove those... but I think they're all used one way or the other.

I think this is just a cursed saved state blob.

Reply:
> I think this is just a cursed saved state blob.

So it goes.

Not sure what the best way to track these sorts of cases might be (I know floppy is another example of weird saved-state shenanigans), but it would be nice to gradually enumerate the sorts of saved-state warts we would want to fix if we ever found time to re-invest in our saved-state infra (particularly wrt. doing various top-level saved-state "manipulations" / "migrations")